traitcuration/traits/templates/traits/trait_detail.html

```diff
@@ -1,26 +1,82 @@
+{% block title %} Trait Detail {% endblock %}
+{% load trait_filters %} 
 
 {% block content %}
-<div class="uk-container uk-container-xlarge">
-    <h1>Trait details</h1>
-    <ul class="uk-list">
-        <li>Trait name: {{trait.name}}</li>
-        <li>Source records: {% firstof trait.number_of_source_records '0' %}</li>
-        <li>Trait status: {{trait.status}}</li>
-    </ul>
-    <h1>Mapped term details</h1>
-    {% with trait.current_mapping.term_id as term %}
-    <ul class="uk-list">
-        <li>Term: {% firstof term.curie 'None' %}</li>
-        <li>Term label: {% firstof term.label 'None' %}</li>
-        <li>Term status: {% firstof term.status 'None' %}</li>
-    </ul>
-    {% endwith %}
-    
-    
+<aside class="sidebar">
+  <div class="sidebar__wrapper">
+    <div class="sidebar__section">
+      <p class="sidebar__section-title">LAST UPDATED</p>
+      <span class="sidebar__section-text"> {{trait.timestamp_updated}} </span>
+      <hr />
+    </div>
+    <div class="sidebar__section">
+      {% if trait.current_mapping %}
+      <p class="sidebar__section-title">CURATOR</p>
+      <span uk-icon="user" class="uk-margin-small-right"></span>
+      <span class="sidebar__section-text">
+        {{trait.current_mapping.curator}}
+      </span>
+      <hr />
+      {% endif %}
+    </div>
+    <div class="sidebar__section">
+      {% if trait.current_mapping %}
+      <p class="sidebar__section-title">REVIEWED BY</p>
+      {% with trait.current_mapping.review_set.all as reviews %} 
+      {% for review in reviews %}
+      <div class="sidebar__review-section">
+        <span uk-icon="user" class="uk-margin-small-right"></span>
+        <span class="sidebar__section-text"> {{review.reviewer}} </span>
+      </div>
+      {% endfor %}
+      <hr />
+      {% endwith %}{% endif %}
+    </div>
+    <div class="sidebar__section">
+      {% if trait.current_mapping %}
+      <p class="sidebar__section-title">REVIEW STATUS</p>
+      {% if trait.current_mapping.is_reviewed %}
+      <span class="uk-label label--success"> REVIEWED</span>
+      {% else %}
+      <span class="uk-label label--warning"> IN REVIEW </span>
+      {% endif %}
+      <button class="uk-button button--success sidebar__section-review-button">
+        <span uk-icon="check"> REVIEW </span>
+      </button>
+      {% endif %}
+    </div>
+  </div>
+</aside>
+
+<div class="uk-container uk-container-xlarge trait-container">
+  <div class="info-section__wrapper">
+    <div class="info-section__column">
+      <span class="info-section__title">TRAIT NAME</span>
+      <span class="info-section__value">{{trait.name}}</span>
+    </div>
+    <div class="info-section__column">
+      <span class="info-section__title">SOURCE RECORDS</span>
+      <span class="info-section__value"
+        >{% firstof trait.number_of_source_records '0' %}</span
+      >
+    </div>
+    <div class="info-section__column">
+      <span class="info-section__title">TRAIT STATUS</span>
```

Comment:
When a trait name is long but window width is not, something like this happens:

![image](https://user-images.githubusercontent.com/10669118/85572962-5be8fa00-b62d-11ea-9fd9-21cd04126bcf.png)

In this case it would be better to make "trait name" column smaller, and to ensure that "source records" and "trait status" will always fit

Reply:
Yes I will do that. Another thing for really small screens (basically just tablets in portrait mode), I could make it so that the trait status wraps beneath the first row to save some space. What do you think?

Reply:
I'm afraid it may mess with the layout and how the users expect the screen do behave when resizing its window. A trait status is an important marker, and people will expect to see it always in the same place. I think that at this point just changing column margins and re-wrapping trait name into multiple rows should be enough, especially considering that few trait names will be this long.

Reply:
I agree, I will add that

Reply:
So, I did fix it by making the sidebar take up 14% of the page which ensures that the trait column always wraps, and it looks like this
1. On a 1920x1080 screen
![image](https://user-images.githubusercontent.com/36301625/85989226-e8623680-b9f8-11ea-907d-63ff6a47178e.png)
2. On a 1360x768px screen
![image](https://user-images.githubusercontent.com/36301625/85989270-f9ab4300-b9f8-11ea-97df-549efcc08802.png)
3. On a 1200x800px tablet
![image](https://user-images.githubusercontent.com/36301625/85989355-10519a00-b9f9-11ea-9cff-1fe9876e6254.png)

As you can imagine the page will be pretty unusable on anything smaller than a landscape tablet. So I could add some media queries to do something like increase the sidebar width and decrease the font size in screens smaller than 800px if that is a concern. Of course I am not referring to having it work on really small screens such as mobile phones, because that would require major redesigning as we discussed.

Reply:
I can confirm that what we have currently is acceptable for reasonable window width and zoom range of 100-150%, so we may consider this issue resolved. Thanks!